projects/cds/src/cds-models/cds-strategy-request.model.ts

```diff
@@ -4,13 +4,18 @@
  * context
  */
 export interface StrategyRequest {
-  // This property is productCodes in OCC
-  products?: string[];
-  category?: string;
-  facets?: string;
-  date?: Date;
-  site?: string;
-  language?: string;
-  pageNumber?: number;
-  pageSize?: number;
+  queryParams?: {
+    // This property is productCodes in OCC
+    products?: string[];
+    category?: string;
+    facets?: string;
+    date?: Date;
+    site?: string;
+    language?: string;
+    pageNumber?: number;
+    pageSize?: number;
+  };
+  headers?: {
+    'consent-reference'?: string;
```

Comment:
does this need to be enclosed in a ' ' ?

Reply:
It does because of the '-'

Reply:
because you're backend requires this exact property with the `-` right?

Reply:
Like Brian mentioned, `projects/cds/src/profiletag/http-interceptors/consent-reference-interceptor.ts` sets the `x-` prefixed consent reference header. Could that do the job?
If not, is it redundant in that case?

Reply:
As mentioned, these are two different use cases. The one mentioned is for sending requests to OCC url's. The new one is for sending requests to CDS Cloud URL's